Playbooks/Enrich-Sentinel-Incident-HYAS-Insight-Domain-Current-WHOIS/azuredeploy.json

```diff
@@ -0,0 +1,368 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+	"metadata": {
+        "comments": "This playbook uses the HYAS Insight connector to automatically enrich incidents generated by Sentinel with current WHOIS information. You need a valid subscription in order to use the connector and playbook. Learn more about the integration via the https://docs.microsoft.com/en-us/connectors/hyasinsight/ or visit https://www.hyas.com/contact to request a trial key.",
```

Comment:
Here is an example where the locale part of the URL is still present

Reply:
Thank you @sarah-yo , I have updated the links accordingly...